### ISSUE TITLE
Add a Prometheus relabel playground to the docs

### ISSUE THREAD

**Issue by ptodev** [grafana/alloy#1921]
### Request

It would he nice if users can experiment with Prometheus relabeling inside the Alloy docs themselves. This could be something similar to the [Promlabs Relabeler](https://relabeler.promlabs.com/) (which might be part of [Promlens](https://github.com/prometheus/promlens)?) and the proposed [OTTL Playground](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/33747). 

In the future, we could extend this to other components. But for now I think `prometheus.relabel` and `discovery.relabel` are the highes priority ones. We could do something similar for `loki.process` too.

### Use case

The best way to learn is by doing. Such a feature will help users get started since they can see for themselves how things work. It will also help users who are debugging their pipeline and want to make sure it's doing the right thing.

**Comment by ptodev** [comment]
I asked @robbymilo if there is anything specific we should keep in mind if we want to embed it in the website. He said that for now we can develop a POC with any libraries we want, and afterwards we can think how to integrate it in the website in a nice way.

**Comment by wildum** [comment]
I feel like the live debugging feature is already covering this use-case. See the videos: https://www.youtube.com/watch?v=_MbB8IVKMfw&t=2s&ab_channel=Grafana and https://www.youtube.com/watch?v=IRqQEzc0kvA&t=12s&ab_channel=Grafana.

The advantage of live debugging compared to the playground is that it uses directly your data so you can make sure that once you're satisfied with the result, it will stay like this.

**Comment by ptodev** [comment]
There are drawbacks to live debugging:
* You have to run Alloy
* You have to configure it to ingest the sort of data you want to test with. That may not necessarily be the data your real Alloy instances are ingesting at that time, so it's not like you can just open a live debugging session.

The idea of the playground is for people to quickly try things out to see if that component even does what they think it does. It's harder to do this if you have to start a real Alloy instance.

Maybe we can somehow embed the live debugging in the website though? Like a unit test where a user would fill in the inputs to the component, and a mock "live debugging" would show them what the component is doing?

**Comment by wildum** [comment]
True, maintenance of these tools could be a concern but I see the use-case.

Maybe one step further could be to have an Alloy sandbox with a configurable env. Something similar to the one used in this tuto https://grafana.com/docs/loki/latest/send-data/alloy/examples/alloy-otel-logs/#dependencies.
This would also allow us to setup a k8s env.

This would be more work but could also be a nice quick way to validate a config.

**Comment by Aref-Riant** [comment]
Having a preview for relabel configs seems really necessary.
